features/tests_by_week_metrics.feature

```diff
@@ -0,0 +1,193 @@
+@api @metrics
+Feature: Reports by day metrics
```

Comment:
It seems to me that there is much copy-paste evil in this file, including in the description of the feature and all scenario names.